static/templates/message_edit_history.handlebars

```diff
@@ -0,0 +1,18 @@
+{{! Client-side Mustache template for viewing message edit history.}}
+
+<div class="date_row"><span id="timerender2">Earliest</span></div>
+{{#each edited_messages}}
+{{#with this}}
```

Comment:
I'm not very familiar with Handlebars. Why do we need the `{{#with this}}` block?